gulp-tasks/test.js

```diff
@@ -502,6 +503,19 @@ function testMarkdown(filename, contents, options) {
       });
     }
 
+    // Validate listed components
+    do {
```

Comment:
change the syntax - use the `{# wf_blink_component_id: a,b,c #}`. The YAML at the top of the page is used by DevSite and this may break the way DevSite parses a page.